modules/estadistica/routes/turnos_prestaciones.ts

```diff
@@ -1,41 +1,56 @@
 import * as express from 'express';
 import * as mongoose from 'mongoose';
+import * as moment from 'moment';
 import { Auth } from '../../../auth/auth.class';
 import * as agendaController from './../controller/procesarAgendas';
 import * as fueraDeAgendaController from './../controller/procesarFueraDeAgenda';
+import * as turnos_prestacionesConstroller from './../controller/turnos_prestacionesController';
 
 
 const router = express.Router();
 
 router.get('/turnos_prestaciones', async (req, res, next) => {
-
     if (!Auth.check(req, 'turnosPrestaciones:buscar')) {
         return next(403);
     }
 
-    const parametros = {
-        organizacion: new mongoose.Types.ObjectId(Auth.getOrganization(req)),
-        fechaDesde: req.query.fechaDesde,
-        fechaHasta: req.query.fechaHasta,
-        prestacion: req.query.prestacion,
-        estado: req.query.estado,
-        profesional: req.query.idProfesional,
-        financiador: req.query.financiador,
-        estadoFacturacion: req.query.estadoFacturacion,
-        documento: req.query.documento,
-        ambito: req.query.ambito
-    };
     try {
-        // Procesa los turnos aplicando los filtros
-        const _turnos = parametros?.ambito === 'internacion' ? [] : agendaController.procesar(parametros);
-        // Procesa las prestaciones fuera de agenda
-        const _prestaciones = fueraDeAgendaController.procesar(parametros);
-        const [turnos, prestaciones] = await Promise.all([_turnos, _prestaciones]);
-        const resultado: any = turnos.concat(prestaciones);
+        const resultado = await turnos_prestacionesConstroller.armarListado(req.query);
         res.json(resultado);
     } catch (error) {
         return next(error);
     }
 });
 
+router.post('/turnos_prestaciones/csv', async (req, res, next) => {
+
+    if (!Auth.check(req, 'turnosPrestaciones:buscar')) {
+        return next(403);
+    }
+
+    try {
+        const csv = require('fast-csv');
+        const resultado = await turnos_prestacionesConstroller.armarListado(req.body);
+        res.set('Content-Type', 'text/csv');
+        res.setHeader('Content-disposition', 'attachment; filename=listado.csv');
+        csv
+            .write(resultado, {
+                headers: true, transform: (row) => {
+                    return {
+                        Fecha: (row.fecha) ? moment(row.fecha).format('DD-MM-YYYY') : '',
```

Comment:
Es un detalle, pero de esta forma se puede ordenar la columna.
```suggestion
                        Fecha: (row.fecha) ? moment(row.fecha).format('YYYY-MM-DD') : '',
```